lib/gui/os/notification.js

```diff
@@ -30,19 +30,21 @@ const electron = require('electron');
  *
  * @param {String} title - notification title
  * @param {String} body - notification body
+ * @param {String} icon - supported icon path
  * @returns {Object} HTML5 notification instance
  *
  * @example
- * notification.send('Hello', 'Foo Bar Bar');
+ * notification.send('Hello', 'Foo Bar Bar', 'icon.png');
  */
-exports.send = (title, body) => {
+exports.send = (title, body, icon) => {
```

Comment:
Can we take an options object, which optionally may contain an `icon` property instead?